### PR TITLE
Final first pass

### PR DESCRIPTION
**Pull request by armenianarkanian**

### CODE REVIEW

**Comment by armenianarkanian** [comment]
How can i delete the posts directory?

**Comment by chrpng** [comment]
> How can i delete the posts directory?

The error is probably from /scripts/gen-rss.js attempting to access the now non-existent posts directory.

File: pages/Professional.mdx [hunk 1]
```diff
@@ -4,41 +4,81 @@ title: Professional
 date: 2023-01-17
 ---
 
+import Image from 'next/image'
```
Comment (chrpng) [inline pages/Professional.mdx:7]: Added import

File: scripts/gen-rss.js [hunk 2]
```diff
@@ -10,28 +10,6 @@ async function generate() {
...
-        title: frontmatter.data.title,
-        url: '/posts/' + name.replace(/\.mdx?/, ''),
-        date: frontmatter.data.date,
-        description: frontmatter.data.description,
-        categories: frontmatter.data.tag.split(', '),
-        author: frontmatter.data.author,
-      })
-    })
-  )
-
```
Comment (chrpng) [inline scripts/gen-rss.js:34]: Deleted post generation

**Pull request merged**